src/routes/+page.md

```diff
@@ -105,7 +106,7 @@ To use it, drop a `<PaymentElement>` component in your form:
 ```html
 <form on:submit|preventDefault="{submit}">
   <Elements {stripe} {clientSecret} bind:elements>
-    <PaymentElement options={...} />
+    <PaymentElement options="{...}" />
```

Comment:
If possible, let's keep linting changes as a separate PR.

Reply:
Sorry, must've autoformatted on save

Reply:
No worries! Happens to me all the time too 😅